tasks/sep/032-op-pectra-defence/README.md

```diff
@@ -0,0 +1,44 @@
+# ProxyAdminOwner - Set Dispute Game Implementation
+
+Status: READY TO SIGN
+
+## Objective
+
+This task updates deploys new dispute games with upgrade prestate hashes on OP Sepolia and Sony Minato.
+
+This updated prestate hash corresponds with the hardfork to support the L1 Pectra upgrade.
+
+## Input data derivation:
+
+The OPPrestateUpdater contract was deployed in advance to [0x8cB886373DC01CA0463E4d8DED639467Ff0989c2](https://sepolia.etherscan.io/address/0x8cB886373DC01CA0463E4d8DED639467Ff0989c2).
+
+The input to the `updatePrestate()` function is an array of the following struct:
+
+```solidity
+  struct OpChainConfig {
+        ISystemConfig systemConfigProxy;
+        IProxyAdmin proxyAdmin;
+        Claim absolutePrestate;
+    }
+```
+
+The prestate hash used is: `0x035ac388b5cb22acf52a2063cfde108d09b1888655d21f02f595f9c3ea6cbdcd`.
+Which was taken from [op-program/prestates/releases.json](https://github.com/ethereum-optimism/optimism/blob/8d0dd96e494b2ba154587877351e87788336a4ec/op-program/prestates/releases.json#L9).
```

Comment:
@Inphi @ajsutton can you confirm this is a good prestate to use for OP Sepolia?

Reply:
Ok so the link on L26 here shows us that this prestate exists in op-program 1.5.0-rc.2. [This gist](https://gist.github.com/ajsutton/8b11c1faf54b451c836d0b4f8dcb6846) from Adrian, and the comment, confirms that Uni, Base, Ink, and OP can all use this prestate for both mainnet and sepolia

Reply:
Resolving on that basis.